Doc/c-api/buffer.rst

```diff
@@ -473,7 +473,7 @@ Buffer-related functions
    (*order* is ``'A'``).  Return ``0`` otherwise.
 
 
-.. c:function:: void PyBuffer_FillContiguousStrides(int ndim, Py_ssize_t *shape, Py_ssize_t *strides, Py_ssize_t itemsize, char order)
+.. c:function:: void PyBuffer_FillContiguousStrides(int ndim, Py_ssize_t *shape, Py_ssize_t *strides, int itemsize, char order)
```

Comment:
First parameter of this function is named `ndims`:

https://github.com/python/cpython/blob/cd195e2a7ac5c9b2574d5462752b7939641de4a9/Include/abstract.h#L559-L563

I'd say go ahead and rename `ndim` to `ndims`.

Reply:
Thanks for pointing that out. Also, the last parameter is "char fort" rather than "char order" in documentation. Do you want me to change that as well ?

Reply:
No, `order` should stay as-is since it's more readable. It was renamed from `fortran` to `order` in https://github.com/python/cpython/commit/9a2d99e28a5c2989b2db4023acae4f550885f2ef#diff-77786d8cf2602a6684bccbf242e5fc1eR457